README.md

```diff
@@ -44,7 +44,7 @@ You will find the smart contracts in the `/contracts` directory. Feel free to pl
 Once the [npm package](https://www.npmjs.com/package/@solarity/solidity-lib) is installed, one can use the library just like that:
 
 ```solidity
```

Comment:
Let's use a different example. Maybe a registry contract with CMT and ZKP verification function.